### ISSUE TITLE
Unable to connect to server with jet boots / bml

### ISSUE THREAD

**Issue by DigitalData** [Crimix/jetboots#30]
## Version Info
MC 1.20.1
Forge 47.2.17
Jet Boots (latest version)

## Error
I wanted to add Jet Boots to a mod pack, however, I am unable to connect to a server I am hosting. The connectivity mod returns the following error:
```
io.netty.handler.codec.DecoderException: io.netty.handler.codec.EncoderException: java.io.UTFDataFormatException: malformed input around byte 57
```

## Why I think it's BML / Jet Boots
1. Remove `bml` and `jet boots` mods from server and client.
2. Restart server and client.
3. Connection works.

## My Theory
_(lol feel free to ignore)_
- This may be a bml error, not a jet boots error. Happy to create an issue there but didn't feel the need to test that, since they're both your mods.
- I don't think it's a compatibility issue with [Connectivity](https://www.curseforge.com/minecraft/mc-mods/connectivity) as the mod's functionality wouldn't be affecting just the one "new items" mod.
- Ignorant research turns up the following extremely old forum post: https://forums.minecraftforge.net/topic/90713-1152-javaioutfdataformatexception-malformed-input-around-byte-1/
  - One now-deleted response (ffs it's like decade-old stack overflow forums) says it's caused by some differences in serializer read and write.

> read and write in your serializer are not symmetrical.
> 
> In read you first read the ingredient then the stack. In write you first write the stack, then the ingredient.

**Comment by Crimix** [comment]
Sorry about the delay of updating my mods, I will try to have a look at it, both I am also thinking of switching to NeoForge

**Comment by DigitalData** [comment]
No stress. It's looking like NeoForge is the logical next step for everyone. I think for 1.20.1 NeoForge and Forge mods are compatible with either modloader. At least that's the case for Immersive Engineering, which is now fully NeoForge.

**Comment by Crimix** [comment]
Sorry it took so lon got fix the issue, but thanks for reporting it.
I forgot about it and spent 1 hour trying to figure out which of my custom mods made my pack crash on server join.

It is fixed in version 1.2.6
